packages/components/src/dimension-control/sizes.ts

```diff
@@ -12,16 +12,19 @@
  */
 import { _x } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+import type { Size } from './types';
+
 /**
  * Finds the correct size object from the provided sizes
  * table by size slug (eg: `medium`)
  *
- * @param {Array}  sizes containing objects for each size definition.
- * @param {string} slug  a string representation of the size (eg: `medium`).
- *
- * @return {Object} the matching size definition.
+ * @param  sizes containing objects for each size definition.
```

Comment:
There seems to be a CI failure related to the alignment of JSDoc comments in this file

```
 error  Expected JSDoc block lines to be aligned  jsdoc/check-line-alignment
```